### PR TITLE
.*: fix build fail in  linux amd64

### LINKED ISSUE #6022: build fail in linux amd64
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 2. What did you expect to see? (Required)
```
[2022-09-25T15:02:36.770Z] FAILED: dbms/src/Flash/CMakeFiles/flash_service.dir/Coprocessor/DAGStorageInterpreter.cpp.o 

[2022-09-25T15:02:36.770Z] /usr/local/bin/clang++ -DBOOST_BIND_GLOBAL_PLACEHOLDERS -DBOOST_CONTEXT_EXPORT="" -DBOOST_CONTEXT_NO_LIB -DBOOST_CONTEXT_STATIC_LINK -DBOOST_FIBER_NO_LIB -DBOOST_FIBER_STATIC_LINK -DBOOST_NO_CXX98_FUNCTION_BASE -DBOOST_SYSTEM_NO_DEPRECATED -DCARES_STATICLIB -DLZ4_DISABLE_DEPRECATE_WARNINGS=1 -DPOCO_STATIC -DPOCO_UNBUNDLED_ZLIB -DSTACK_LINE_READER_BUFFER_SIZE=1024 -DTIFLASH_COMPILER_VPCLMULQDQ_SUPPORT=1 -DTIFLASH_ENABLE_AVX512_SUPPORT=1 -DTIFLASH_ENABLE_AVX_SUPPORT=1 -DTIFLASH_SOURCE_PREFIX=\"/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics\" -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/dbms/src -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/tiflash-proxy/raftstore-proxy/ffi/src -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/release-centos7-llvm/build-release/dbms/src -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/double-conversion -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/boost -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/xxHash -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/libpcg-random/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/libcityhash/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/libs/libcommon/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/release-centos7-llvm/build-release/libs/libcommon/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/libs/libpocoext/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/poco/Data/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/poco/Foundation/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/zlib-ng -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/release-centos7-llvm/build-release/contrib/zlib-ng -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/poco/Util/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/poco/XML/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/poco/JSON/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/poco/Net/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/cctz/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/jemalloc/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/jemalloc-cmake/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/cpu_features/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/libcpuid/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/lz4/lib -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/zstd/lib -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/client-c/third_party/libfiu/libfiu -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/prometheus-cpp/core/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/release-centos7-llvm/build-release/contrib/prometheus-cpp-cmake/core/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/prometheus-cpp/push/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/release-centos7-llvm/build-release/contrib/prometheus-cpp-cmake/push/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/prometheus-cpp/pull/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/release-centos7-llvm/build-release/contrib/prometheus-cpp-cmake/pull/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/cpptoml -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/magic_enum/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/poco/NetSSL_OpenSSL/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/poco/Crypto/include -I/usr/local/opt/openssl/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/xxHash/cmake_unofficial/.. -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/release-centos7-llvm/build-release/contrib/kvproto/cpp -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/release-centos7-llvm/build-release/contrib/kvproto/cpp/kvproto -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/grpc/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/abseil-cpp -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/client-c/include -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/release-centos7-llvm/build-release/contrib/grpc/third_party/cares/cares -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/grpc/third_party/cares/cares -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/release-centos7-llvm/build-release/contrib/tipb/cpp -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/release-centos7-llvm/build-release/dbms/src/Storages/DeltaMerge/File/dtpb -I/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/libbtrie/include -isystem /home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/fmtlib-cmake/../fmtlib/include -isystem /home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/libs/libsymbolization/include -isystem /home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/protobuf/src -isystem /home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/contrib/re2 -isystem /home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/release-centos7-llvm/build-release/contrib/re2-cmake -D__GLIBCXX_BITSIZE_INT_N_0=128 -D__GLIBCXX_TYPE_INT_N_0=__int128 -flto=thin -fvisibility=hidden -fvisibility-inlines-hidden -fwhole-program-vtables -fsplit-lto-unit  -D_GLIBCXX_USE_CXX11_ABI=1 -pipe -msse4.1 -msse4.2 -mpopcnt  -fno-omit-frame-pointer  -Wall -Wno-unused-command-line-argument  -Wnon-virtual-dtor  -stdlib=libc++ -Wextra -Werror -O2 -g -DNDEBUG -O3  -fPIC -DDUMMY_BACKTRACE -pthread -std=gnu++17 -Winvalid-pch -Xclang -include-pch -Xclang /home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/release-centos7-llvm/build-release/dbms/src/Flash/CMakeFiles/flash_service.dir/cmake_pch.hxx.pch -Xclang -include -Xclang /home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/release-centos7-llvm/build-release/dbms/src/Flash/CMakeFiles/flash_service.dir/cmake_pch.hxx -MD -MT dbms/src/Flash/CMakeFiles/flash_service.dir/Coprocessor/DAGStorageInterpreter.cpp.o -MF dbms/src/Flash/CMakeFiles/flash_service.dir/Coprocessor/DAGStorageInterpreter.cpp.o.d -o dbms/src/Flash/CMakeFiles/flash_service.dir/Coprocessor/DAGStorageInterpreter.cpp.o -c /home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp

[2022-09-25T15:02:36.772Z] /home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/tics/dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp:242:58: error: unused parameter 'query_info' [-Werror,-Wunused-parameter]

[2022-09-25T15:02:36.772Z] void injectFailPointForLocalRead(const SelectQueryInfo & query_info)

[2022-09-25T15:02:36.772Z]                                                          ^

[2022-09-25T15:02:36.772Z] 1 error generated.

[2022-09-25T15:02:36.772Z] [3140/3246] Building CXX object dbms/src/Functions/CMakeFiles/clickhouse_functions.dir/divide.cpp.o

[2022-09-25T15:02:36.772Z] ninja: build stopped: subcommand failed.
```

### 3. What did you see instead (Required)
build success

### 4. What is your TiFlash version? (Required)
master

### PR DESCRIPTION
**Pull request by SeaRise**
### What problem does this PR solve?

Issue Number: close #6022

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by JaySon-Huang — APPROVED** [review]

File: dbms/src/TestUtils/TiFlashTestEnv.cpp [hunk 1]
```diff
@@ -83,8 +83,11 @@ void TiFlashTestEnv::addGlobalContext(Strings testdata_path, PageStorageRunMode
     KeyManagerPtr key_manager = std::make_shared<MockKeyManager>(false);
     global_context->initializeFileProvider(key_manager, false);
 
+
+#ifndef NDEBUG
     // do not need to load config file in unit test, just set `is_config_loaded = true` to skip checkIsConfigLoaded
     global_context->is_config_loaded = true;
+#endif
```
Comment (SeaRise) [inline dbms/src/TestUtils/TiFlashTestEnv.cpp:90]: fix https://github.com/pingcap/tiflash/pull/5988#discussion_r979532862

**Review by solotzg — APPROVED** [review]

**Pull request merged**